modules/oc-mirror-installing-plugin.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="installation-oc-mirror-installing-plugin_{context}"]
+= Installing the oc-mirror OpenShift CLI plug-in
+
+To use the oc-mirror OpenShift CLI plug-in to mirror registry images, you must install the plug-in. If you are mirroring image sets in a fully disconnected environment, ensure that you install the oc-mirror plug-in on all required hosts.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Download the oc-mirror CLI plug-in.
+
+.. Navigate to the link:https://console.redhat.com/openshift/downloads[Downloads] page of the {console-redhat-com}.
+
+.. Under the *Command-line interface (CLI) tools* section, click *Download* for *OpenShift Client (oc) mirror plugin* and save the file.
```

Comment:
From now , could not see the download for oc plugin

Reply:
Right, this isn't available yet on this page - I assume it will not be there until 4.10 GA.

@afflom @jpower432 Can you help @zhouying7780 with if there is a way to test this now, without there being the binary available on the live downloads page yet?

Reply:
Sorry that I missed this tag. I've posted a one liner for Ying in slack to retrieve the oc-mirror from the release payload.